phpBB/includes/acp/acp_search.php

```diff
@@ -392,6 +392,13 @@ private function index_action(string $id, string $mode, string $action): void
 			}
 		}
 
+		// Start displaying progress on first submit
```

Comment:
The best way to know if it's the first time you've entered here, I think it's putting it a few lines above, and you don't have to look at the post variables